Calculator/CalculatorBrain.swift

```diff
@@ -18,14 +26,103 @@ class CalculatorBrain {
     var operatorType: OperatorType?
     
     func addOperandDigit(_ digit: String) -> String {
-        
+        if operatorType == nil {
+            if digit == ".", operand1String.contains(".") {
+                return operand1String
+            }
+            operand1String.append(digit)
+            return operand1String
+        } else {
+            if digit == ".", operand2String.contains(".") {
+                return operand2String
+            }
+            operand2String.append(digit)
+            return operand2String
+        }
```

Comment:
Another way to add Strings (append) to the end of Strings is to use +=. So it would look like this:

`operand2String += digit`